Makefile.am

```diff
@@ -84,9 +84,9 @@ CONTAINER_ENGINE ?= podman
 CONTAINER_BUILD_ARGS ?=
 CONTAINER_TEST_ARGS ?=
 
-# ci-tasks container
-CI_TASKS_DOCKERFILE ?= $(srcdir)/dockerfile/ci-tasks
-CI_TASKS_NAME ?= anaconda/ci-tasks
+# anaconda-ci container
+CI_TASKS_DOCKERFILE ?= $(srcdir)/dockerfile/anaconda-ci
+CI_TASKS_NAME ?= rhinstaller/anaconda-ci
```

Comment:
The variables should be renamed as well for consistency. You could probably ignore the "anaconda-" prefix though, i. e. `CI_DOCKERFILE` seems fine? `</bikeshedding>`

Reply:
I did not want to break existing stuff but it's still early so let's rename even these.